docs/source/en/model_doc/superpoint.md

```diff
@@ -86,23 +86,29 @@ model = SuperPointForKeypointDetection.from_pretrained("magic-leap-community/sup
 
 inputs = processor(images, return_tensors="pt")
 outputs = model(**inputs)
+image_sizes = [(image[1], image[0]) for image in images]
```

Comment:
it should be either `image.shape[1]` or `image.size[1]` I think and not directly image itself

it would've been directly to have `image.size` from convenience standpoint IMHO i.e. each tuple would've been `image.size` and not indexed

Reply:
Good catch, it's a typo from a wrong copy paste, should be fixed now. Also, the fact that it is `tuple(image.size[1], image.size[0])` is because we require `(h,w)` in input (I guess it's standard from what I've seen in other post process methods in the repo) but `image.size` returns `(w, h)`